radicle-surf/src/object/blob.rs

```diff
@@ -83,53 +81,34 @@ impl Serialize for Blob {
 }
 
 /// Variants of blob content.
-#[derive(PartialEq, Eq)]
-pub enum BlobContent {
-    /// Content is plain text and can be passed as a string.
-    Plain(String),
-    /// Content is binary and needs special treatment.
-    Binary(Vec<u8>),
+pub struct BlobContent<'a> {
+    git2_blob: git2::Blob<'a>,
```

Comment:
We tend to name these kinds of things `inner`:
```suggestion
    inner: git2::Blob<'a>,
```